### PR TITLE
ISV-8217-new-reports-to-seller-raw-data

### PR DESCRIPTION
**Pull request by davi17g**

### CODE REVIEW

File: vast.go [hunk 1]
```diff
@@ -96,6 +102,30 @@ func (v *VAST) RemovePixels() {
 	}
 }
 
+func (v *VAST) GetCreativeType() string {
```
Comment (hadassel) [inline vast.go]: This method should return an error instead of an empty string when there's no match.

File: vast.go [hunk 2]
```diff
@@ -96,6 +102,30 @@ func (v *VAST) RemovePixels() {
...
+	}
+	inLine := ad.InLine
+	if inLine == nil {
+		return ""
+	}
+	creatives := inLine.Creatives
+	if len(creatives) == 0 {
+		return ""
+	}
+	if creatives[0].APIFramework == "" {
```
Comment (hadassel) [inline vast.go]: Are you sure this is a correct logic?

**Review by hadassel — CHANGES_REQUESTED** [review]

File: vast.go (hunk 2 above)
Comment (davi17g) [reply vast.go]: yes. I discussed about this with Aviram

File: vast.go (hunk 1 above)
Comment (davi17g) [reply vast.go]: Fixed

**Pull request merged**